scripts/k6/stream-api/static.js

```diff
@@ -9,13 +9,13 @@ export function setup() {
       exportId: "test-export",
       name: "test-export",
       mapping: {
-        tableId: "in.c-buffer-static.data",
+        tableId: "in.c-stream-static.data",
         columns: [
-          {type: "id", name: "id"},
-          {type: "datetime", name: "datetime"},
-          {type: "ip", name: "ip"},
-          {type: "body", name: "body"},
-          {type: "headers", name: "headers"},
+          { type: "id", name: "id" },
```

Comment:
I think, we should use uuid-v7 type, because have changed it in the migrate script.

https://github.com/keboola/keboola-as-code/blob/39848ff0a2107ac192517a787677ab05d61f0fb1/internal/pkg/service/stream/migrate/core/sink.go#L86

Reply:
✔️